lib/multigrid.cpp

```diff
@@ -15,15 +15,13 @@ namespace quda
 
   static bool debug = false;
 
-  MG::MG(MGParam &param, TimeProfile &profile_global) :
-    Solver(*param.matResidual, *param.matSmooth, *param.matSmoothSloppy, *param.matSmoothSloppy, param, profile),
+  MG::MG(MGParam &param) :
+    Solver(*param.matResidual, *param.matSmooth, *param.matSmoothSloppy, *param.matSmoothSloppy, param),
     param(param),
     transfer(0),
     resetTransfer(false),
     presmoother(nullptr),
     postsmoother(nullptr),
-    profile_global(profile_global),
-    profile("MG level " + std::to_string(param.level), false),
```

Comment:
Does this mean there are no longer separate profiler instances for each MG level? Or am I missing something? The breakdowns for each level were in principle nice (though sometimes they seemed a bit quirky).

Reply:
Yes, they are gone.  They were indeed very quirky, and didn't report the coarse level properly previously.

I introduced them when I first created the MG solver in QUDA to get a handle of coarse grid time, but their use is somewhat redundant now (given the kernel profile for example).  I might re-introduce them in another form at some point, once more thought has gone into evolving the `TimeProfile`.